_posts/2021-02-23-new-users.md

```diff
@@ -0,0 +1,39 @@
+---
+site: freiburg
+title: We want to welcome all new Galaxy users!
+tags:
+  - data
+  - tools
+  - training
+supporters:
+  - denbi
+  - elixir
+---
+
+For all new Galaxy users, we want to summarize briefly our service.
+
+__With Galaxy we offer you free, automatized analyses of (-omics) data for your lab!__
+
+We would like to invite you to use our free data analysis gateway Galaxy for your daily work. To use Galaxy, you do not need any (bio)informatics skill or to install software. All you need is a browser such as Chrome or Firefox.
+
+Our Galaxy platform provides >2500 data analysis tools for e.g. __sequence analyses, genome annotation, epigenetics, metabolomics, proteomics, metagenomics, statistics and machine learning, and numerous visualizations__.
```

Comment:
link to https://galaxyproject.eu/tools ?

Reply:
added